src/js/techreport/utils/data.js

```diff
@@ -62,11 +62,13 @@ const parsePageWeightData = (metric, date) => {
       ...submetric,
       desktop: {
         ...submetric.desktop,
+        median_bytes_formatted: submetric.desktop.median_bytes > 1024 ? `${Math.round(submetric.desktop.median_bytes / 1024)} KB` : submetric.desktop.median_bytes > 1048576 ? `${Math.round(submetric.desktop.median_bytes / 1048576)} MB` : `${submetric.desktop.median_bytes} bytes`,
         client: 'desktop',
         date: date,
       },
       mobile: {
         ...submetric.mobile,
+        median_bytes_formatted: submetric.mobile.median_bytes > 1024 ? `${Math.round(submetric.mobile.median_bytes / 1024)} KB` : submetric.mobile.median_bytes > 1048576 ? `${Math.round(submetric.mobile.median_bytes / 1048576)} MB` : `${submetric.mobile.median_bytes} bytes`,
```

Comment:
Same comment. (should this conversion be moved to a reusable function?)

Reply:
Good point, will move to a function in the utils 👍🏻